test/index.test.ts

```diff
@@ -361,16 +366,29 @@ describe('HipThrusTS', () => {
         const returnAssignableToCorrectShouldFail: ReturnAssignableToCorrect = false;
         // @ts-expect-error
         const returnAssignableFromCorrectShouldFail: ReturnAssignableFromCorrect = false;
+
+        const pipedAtoBCAttachDataReturnValues = await pipedAtoBC.attachData({
+          a: aPassedIn,
+        });
+
+        expect(pipedAtoBCAttachDataReturnValues).to.has.property('b');
+        expect(pipedAtoBCAttachDataReturnValues.b).to.be.equal(4);
+        expect(pipedAtoBCAttachDataReturnValues).to.has.property('c');
+        expect(pipedAtoBCAttachDataReturnValues.c).to.be.equal(4);
       });
-      it('attaches right sync output instead of left sync output if right output sync transform left output type', () => {
+      it('attaches right sync output instead of left sync output if right output sync transform left output type', async () => {
+        const aPassedIn = 5;
+        const bPassedIn = 'some string';
         const left = {
           attachData(context: { a: number }) {
+            expect(context.a).to.be.equal(aPassedIn);
             return { b: 'some string' };
           },
         };
 
         const rightTransformLeftOutputType = {
           attachData(context: { b: string }) {
+            expect(context.b).to.be.equal(bPassedIn);
             return { b: 5 };
           },
```

Comment:
- [ ] also use a variable "bReturned" or something for 5.

Reply:
done